Makefile

```diff
@@ -7,4 +7,4 @@ protos: go_setup
 go_run:
 	go run examples/go/main.go
 
-.PHONY: protos go_setup go_run
```

Comment:
Let's leave a trailing newline. I suspect this PR doesn't need to modify this file at all unless we're adding a python example.

Reply:
+1 to leaving the trailing newline. But I do think this PR should add a `make -C protos python` command under the `protos` target.

Reply:
Ah I forgot that target was in this file, my bad.